posts/Davide/2016-10-10-hibernate-ogm-5-1-alpha-and-5-0-2-released.adoc

```diff
@@ -0,0 +1,119 @@
+= "The first hibernate OGM 5.1 Alpha and 5.0 maintenance release"
+Davide D'Alto
+:awestruct-tags: [ "Hibernate OGM", "Releases" ]
+:awestruct-layout: blog-post
+---
+
+Good news!
+
+We released http://hibernate.org/ogm/[Hibernate OGM] 5.1 Alpha1 and 5.0.2 Final!
+
+== What's new?
+
+=== Hibernate OGM 5.0.2.Final
+Hibernate OGM 5.0.2.Final supports now http://www.mongodb.com[MongoDB] 3.2 and it's stll
+compatible backward compatible with Hibernate OGM 5.0.1.Final.
+
+If you need to upgrade from a previous version, you can find help on the
```

Comment:
s/previous version/pre 5.0 version/ ? It would be more clear as people doesn't need to check for the migration notes if they are upgrading from 5.0.

s/help on the/help in the/ ?